package.json

```diff
@@ -127,7 +127,7 @@
     "connect": "^3.7.0",
     "jsonata": "^2.0.6",
     "sequelize": "^6.37.7",
-    "sqlite3": "github:AppThreat/node-sqlite3#d9bd20f3991ced9c48bf339a77b0cbb5605db8e9"
+    "sqlite3": "npm:@appthreat/sqlite3@^6.0.2"
```

Comment:
Libraries such as Sequelize expect the import to be [called](https://github.com/sequelize/sequelize/blob/5b7c8015c907b0dfd878ca4a9f28343a3392c307/src/dialects/sqlite/connection-manager.js#L24) `sqlite3`. However, our forked package is published as `@appthreat/sqlite3`. So we make use of pnpm [aliases](https://pnpm.io/aliases) feature to force the name `sqlite3` for our custom fork!